### DISCUSSION TITLE
Multiple users reported having NVDA appear on their desktops yesterday, without having installed it.

### DISCUSSION THREAD

**Discussion by InDublin** [nvaccess/nvda#16792] [Q&A] [answered]
I checked one laptop, I can see the created time for the files, but no installation events in event viewer application log. Some are domain joined, others are not. Is this app typically installed as part of another popular installation? I've no idea how it came to be installed on several Windows 10-11 systems yesterday afternoon. Any suggestions welcome :)

**Comment by XLTechie** [comment]
Are you certain that the thing installed as "NVDA", is actually the NV Access screen reader NVDA?

Its executable (and some others), should be signed by NV Access Limited.
(Right click nvda.exe, more options, properties, digital signatures.)

**Reply by InDublin** [reply, marked as answer]
Hi and thanks for the quick response. It turns out we use a utility called Ninite to deploy and update software. It had an issue where is saw several systems as duplicates. So, one system that had this software installed, prompted for the installation/updates in 20-30 systems. So, a separate issue.

**Comment by XLTechie** [comment]
Okay, thanks for the follow-up. Closing as not NVDA's responsibility.
